### DISCUSSION TITLE
Create Initials based on First and last names.

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8553] [Q&A] [answered]
I have User Profile extended with First and Last name.
Both are text fields.

Now i added additional one **Initials** and i would like Orchard store the Initials based on the First and last names once Content Item Created / Saved or Updated.

How to update Content Items i  know, 
i am wondering how to get the Save / Update event?

Thanks!

**Comment by jtkech** [comment, marked as answer]
@vitalybrandes 

Through a content handler, by creating a class derived from `ContentHandlerBase` and registering it as an `IContentHandler` and as a scoped service. Then override the events you want e.g. `UpdatedAsync()`, `PublishedAsync()`, `DraftSavedAsync()`, and so on
